R/us-state-populations.R

```diff
@@ -16,11 +16,13 @@
 #'   data.
 #'
 #'   }
-#' @format A data frame with 983 observations of 4 variables.
+#' @format A data frame with 1034 observations of 4 variables.
```

Comment:
Since `devtools::document()` loads the package would it work to do this:

```suggestion
#' @format A data frame with `r nrow(us_state_populations)` observations of 4 variables.
```

Reply:
Thanks for the suggestion! I'll work on that.

Reply:
So the code above doesn't work. And I haven't seen any examples where the number of rows is rendered dynamically: https://github.com/search?q=%22A+data+frame+with%22+%22rows%22+%22variables%22+owner%3Acran+language%3AR&type=code&ref=advsearch. So I've decided to hard code it for now. Thanks for the suggestions tho!

Reply:
Thanks for trying!